clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -51,7 +62,17 @@ const ScanResultsForm = () => {
   );
 
   const handleSubmit = (values: FormValues) => {
-    dispatch(chooseSystemsForReview(values.selectedKeys));
+    if (systems.length > values.selectedKeys.length) {
```

Comment:
I was trying to understand what this was doing, and I think I realized [the ticket](https://github.com/ethyca/fides/issues/598) was vague was asking for something else.

Here's an overview of the flow
1. Submit credentials, scanning happens
2. `ScanResultsForm` shows a table of systems the scan found.
3. The user selects some of those systems, moves onto review.
    - _This PR_ would warn them if they didn't select _all_ the systems.
4. Each system is put through the "review" steps where the user adds more info to the system
5. With [my PR](https://github.com/ethyca/fides/pull/1454/files#diff-7a1370b14e0de70c4fccb724e8dc77d44336b5f57a6590b9644d6f69b49a1210L100), each system has "success" page that indicates how many more systems they have to review.
    - My interpretation of the ticket is that it is referring to _this_ step: if the user hits "cancel" it would warn them that they haven't finished reviewing the systems they had selected.
6. When all systems have been reviewed, the final page is shown. (Just says "datamap" at the moment.)

Reply:
Agreed that the ticket is a bit vague. When I asked design for more clarity on where they think this modal goes or what it's referring to, Adam made a design in Figma for me to follow. [Here's the Slack thread ](https://ethyca.slack.com/archives/C0114BAK160/p1666032070085429)for reference.